### ISSUE TITLE
Dropdown cannot be invoked as per instructions from Voice Over

### ISSUE THREAD

**Issue by sebbe33** [OfficeDev/office-ui-fabric-react#4386]
### Bug Report

- __Package version(s)__: 5.70.0
- __Browser and OS versions__: Safari Version 11.0.3 (12604.5.6.1.1), MacOS Sierra 10.12.6

#### Priorities and help requested (not applicable if asking question):
Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: N/A

#### Describe the issue:
When using Voice Over on MacOS and applying focus to the Dropdown component, VO will announce ‘Press Ctrl + Option + Space to invoke the drop down’. But on pressing these keys, drop down can’t be invoked.

Note: This issue does not appear in Chrome on MacOS.

#### Steps to reproduce:
* Enable Voice Over
* Open Safari and go to https://developer.microsoft.com/en-us/fabric#/components/dropdown
* Tab to a Dropdown component
* Follow the instructions of VO (Ctrl + Option + Space)

#### Actual behavior:
Nothing happens when pressing 'Ctrl + Option + Space' - The drop down list does not open.

Note: User can be invoked on pressing enter or space.

#### Expected behavior:
The drop down list should open.

Following is a screenshot of the issue. Note that this screenshot is taken on a different OS version and with a different VO shortcut, but the problem is the same.
![screen shot 2018-03-28 at 17 11 14](https://user-images.githubusercontent.com/3909617/38042942-9ff18e02-32ad-11e8-825d-805589d25ae2.png)
